src/components/network-chart/index.css

```diff
@@ -90,14 +80,11 @@
 .networkChartRowItemUriRequired {
   display: inline-block;
   overflow: hidden;
+  max-width: 600px;
+  text-overflow: ellipsis;
+  white-space: nowrap;
 }
 
-.networkChartRowItemUriRequired:nth-child(1) {
-  padding-right: 8px;
-}
-
-/* This makes the URI completely visible on hover */
 .networkChartRowItem:hover .networkChartRowItemUriOptional {
-  max-width: unset;
```

Comment:
As you said on slack, because the URI is displayed in the tooltip, we don't need to make it visible in the chart on hover.